rust/kernel/ioctl.rs

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-2.0
+#![allow(non_snake_case)]
+
+//! ioctl() number definitions
```

Comment:
```suggestion
//! `ioctl()` number definitions.
```

Why the parenthesis, by the way?